router/routes/pew.js

```diff
@@ -23,7 +26,10 @@ router.post('/', function(request, response) {
   if (!input || input.length === 0) {
     message.push("now");
   } else {
-    message.push("in", input, "minutes");
+    parse = input.split(' ');
+    minutes = parse[0];
+    game = parse[1];
```

Comment:
Since `game` can be `''` or `undefined`, you should require lodash and do a `_.compact` on `message` before you do the `join` in `slack.send`.

https://lodash.com/docs#compact